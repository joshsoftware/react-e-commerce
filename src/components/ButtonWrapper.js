import React from 'react';
import { Button } from 'reactstrap';
import PropTypes from 'prop-types';
const ButtonWrapper = ({ style, onClick, buttonText, color, disabled, outline, active }) => {
  return (
    <Button
      className={style}
      active={active}
      onClick={onClick}
      color={color}
      outline={outline}
      disabled={disabled}>
      {buttonText}
    </Button>
  );
};

export default ButtonWrapper;

ButtonWrapper.propTypes = {
  active: PropTypes.bool,
  outline: PropTypes.bool,
  color: PropTypes.string,
  disabled: PropTypes.bool,
  style: PropTypes.string,
  onSubmit: PropTypes.func,
  buttonText: PropTypes.string,
  onClick: PropTypes.func
};
