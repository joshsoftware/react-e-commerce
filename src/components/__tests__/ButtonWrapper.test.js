import ButtonWrapper from '../ButtonWrapper';
import { render } from '@testing-library/react';
import React from 'react';

describe('ButtonWrapper Component', () => {
  it('Must return FButtonWrapper Component', () => {
    const { asFragment } = render(<ButtonWrapper />);
    expect(asFragment()).toMatchSnapshot();
  });
});
