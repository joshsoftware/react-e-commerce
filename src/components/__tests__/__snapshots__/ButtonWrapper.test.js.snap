// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonWrapper Component Must return ButtonWrapper Component 1`] = `
<DocumentFragment>
  <button
    class="btn btn-secondary"
  />
</DocumentFragment>
`;
